CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ stage in the `PlannerPipeline` and to generate performance metrics for the indiv
 - Codecov report uploads in GitHub Actions workflow
 - GitHub Actions capability to run on forks
 - Negation overflow caused by minimum INT8
+- Type Inferencer Internal error caused by arithmetic operations with mixed types. 
```

Comment:
nit: the error is more so related to the evaluator's integer overflow checking not the type inferencer